components/transaction/internal/app/command/create-assetrate.go

```diff
@@ -0,0 +1,68 @@
+package command
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	"github.com/LerianStudio/midaz/common"
+	"github.com/LerianStudio/midaz/common/mlog"
+	a "github.com/LerianStudio/midaz/components/transaction/internal/domain/assetrate"
```

Comment:
please, use **ar** for **asset_rate** because we are using **a** for **accounts**, just to normaliza the imports.